packages/x-components/cypress.config.ts

```diff
@@ -31,11 +31,17 @@ export default defineConfig({
     experimentalRunAllSpecs: true
   },
   component: {
+    defaultCommandTimeout: 7000,
+    experimentalSingleTabRunMode: true,
```

Comment:
This flag enables `Cypress` to run all the component tests in a single tab, which comes with better performance and better behavior overall.
It is experimental, but it is working just fine and it seems that, eventually, it will become the default option.